pkg/resource/files.go

```diff
@@ -84,35 +87,27 @@ func (r *Files) Since() string {
 
 // ReadFile provides a way to access values in the JSON data
 func (r *Files) ReadFile(path string) ([]byte, error) {
-	return os.ReadFile(filepath.Join(r.path, filepath.Clean(path)))
+	// Clean the path before passing it to the fs. This also turns "" -> "."
+	// which is important given we don't use "." to represent an empty path
+	file, err := r.fs.Open(filepath.Clean(path))
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Make sure the file is closed after reading the contents.
```suggestion
	}
	defer file.Close()
```